iron-dropdown.html

```diff
@@ -288,16 +288,20 @@
 
         /**
          * Called when the value of `opened` changes.
-         *
-         * @param {boolean} opened True if the dropdown is opened.
+         * Overridden from `IronOverlayBehavior`
          */
-        _openedChanged: function(opened) {
```

Comment:
since a long time by this [bad boy](https://github.com/PolymerElements/iron-overlay-behavior/blob/master/iron-overlay-behavior.html#L246)